packages/loaders/src/Dots.spec.js

```diff
@@ -46,6 +50,60 @@ describe('Dots', () => {
   });
 
   describe('Animation', () => {
+    it('relies on css animation if animated svg is not supported', () => {
+      useCSSSVGAnimation.mockReturnValue(true);
+      const { container } = render(<Dots data-test-id="dots" />);
+
+      act(() => {
+        jest.runOnlyPendingTimers();
+      });
+
+      expect(container.querySelector('g')).toMatchInlineSnapshot(`
+        .c0 {
+          -webkit-animation: dMEWJg 1250ms linear infinite;
+          animation: dMEWJg 1250ms linear infinite;
+        }
+
+        .c1 {
+          -webkit-animation: hiIgQS 1250ms linear infinite;
+          animation: hiIgQS 1250ms linear infinite;
+        }
+
+        .c2 {
+          -webkit-animation: jfCXbz 1250ms linear infinite;
+          animation: jfCXbz 1250ms linear infinite;
```

Comment:
> relies on css animation if animated svg is not supported

It looks like CSS animation is used on Chrome 79 (latest). Does this mean that the Chrome doesn't support animated SVGs? I would have thought it was the older browsers like IE11 that don't support animated SVGs.

<img width="1091" alt="Screen Shot 2020-02-04 at 9 46 49 AM" src="https://user-images.githubusercontent.com/1811365/73771496-5a2ca180-4733-11ea-9f82-56cf41c35ed6.png">

Reply:
This is what styled-components outputs I know edge uses -webkit prefixes for some CSS

Reply:
I made a mistake with the test name. It should be "relies on svg transition attribute if css svg animation is not supported".